consensus/src/dag/dag_store.rs

```diff
@@ -161,6 +174,8 @@ impl Dag {
         self.storage.save_certified_node(&node)?;
         debug!("Added node {}", node.id());
         round_ref[index] = Some(NodeStatus::Unordered(node.clone()));
+        self.payload_manager
+            .prefetch_payload_data(node.payload(), node.metadata().timestamp());
```

Comment:
The crash recover and fetch paths go from here as well, right?

Reply:
it does, also the prefetch needs to tolerate multiple calls as well.